content/docs/rfcs/14/README.md

```diff
@@ -34,183 +38,227 @@ When sending messages over Waku, there are multiple requirements:
+A `WakuMessage` is constituted by the combination of the data payload and attributes that, for example, 
+a *publisher* sends to a `pubsubTopic` and is eventually delivered to *subscribers*.
 
-Waku message attributes are key-value pairs of metadata associated with a message. 
-And the message data payload is the part of the transmitted Waku message that is the actual message information.
-The data payload is also treated as a Waku message attribute for convenience.
+The `WakuMessage` attributes are key-value pairs of metadata associated with a message. 
+The message data payload is part of the transmitted `WakuMessage` that is the actual message information.
+The data payload is also treated as a `WakuMessage` attribute for convenience.
 
-## Message Attributes
+### Message Attributes
 
 * The `payload` attribute MUST contain the message data payload to be sent.
 
-* The `content_topic` attribute MUST specify a string identifier that can be used for content-based filtering.
+* The `contentTopic` attribute MUST specify a string identifier that can be used for content-based filtering,
+as desribed in [23/WAKU2-TOPICS](/spec/23/).
 
-* The `meta` attribute, if present, contains an arbitrary application-specific variable-length byte array with a maximum length limit of 64 bytes.
-This attribute can be utilized to convey supplementary details to various Waku protocols, thereby enabling customized processing based on its contents.
+* The `meta` attribute, if present,
+SHOULD contain an arbitrary application-specific variable-length byte array with a maximum length limit of 64 bytes.
+This attribute can be utilized to convey supplementary details to various Waku protocols, thereby
+enabling customized processing based on its contents.
 
-* The `version` attribute, if present, contains a version number to discriminate different types of payload encryption.
+* The `version` attribute, if present,
+SHOULD contain a version number to discriminate different types of `payload` encryption.
 If omitted, the value SHOULD be interpreted as version 0.
 
-* The `timestamp` attribute, if present, signifies the time at which the message was generated by its sender.
+* The `timestamp` attribute, if present,
+SHOULD signify the time at which the message was generated by its sender.
 This attribute MAY contain the Unix epoch time in nanoseconds.
 If the attribute is omitted, it SHOULD be interpreted as timestamp 0.
 
-* The `ephemeral` attribute, if present, signifies the transient nature of the message.
-For example, an ephemeral message SHOULD not be persisted by the Waku network.
-If this attribute is set to `true`, the message SHOULD be interpreted as ephemeral.
-If, instead, the attribute is omitted or set to `false`, the message SHOULD be interpreted as non-ephemeral.
+* The `ephemeral` attribute, if present,
+signifies the transient nature of the message.
+For example, an ephemeral message SHOULD not be persisted by the [64/WAKU2-NETWORK](/spec/64).
```

Comment:
Perhaps a bit weird to refer to a specific network deployment (RFC64) here. Perhaps the more correct phrasing is more generic:

`SHOULD not be persisted by any Waku nodes`